code/game/machinery/adv_med.dm

```diff
@@ -349,10 +349,10 @@
 			var/obj/item/weapon/paper/P = new /obj/item/weapon/paper(get_turf(target))
 			var/name = occupant ? occupant.name : "Unknown"
 			P.info = "<CENTER><B>Body Scan - [name]</B></CENTER><BR>"
-			P.info += "<b>Time of scan:</b> [worldtime2stationtime(world.time)]<br><br>"
+			P.info += "<b>Time of scan:</b> [stationtime2text(world.time)]<br><br>"
```

Comment:
stationtime2text doesnt' actually have arguments... I'm surprised it didn't error-out but you can just remove the argument in both cases.